tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -148,6 +154,17 @@ NFS Volume Cleanup
     Gather NFS Logs
     Nimbus Cleanup  ${list}
 
+Ping NFS Server
+    ${rc}  ${out}=  Ping Host  ${NFS_READONLY_IP}
+    Log  ${rc}
+    Log  ${out}
+
+Ping VCH
+    ${rc}  ${out}=  Ping Host  %{VCH-IP}
+    Log  ${rc}
+    Log  ${out}
```

Comment:
check that these pings succeed here, then where you call the ping nfs server/vch keyword wrap it in wait until keyword succeeds for about 10 minutes to try to and wait until the network disruption subsides before failing the test